api/src/test/java/org/pmiops/workbench/disk/DiskAdminServiceTest.java

```diff
@@ -136,43 +130,40 @@ public void tearDown() {
 
   @Test
   public void testCheckPersistentDisksNoDisks() {
-    stubDisks(Collections.emptyList());
```

Comment:
old version: needed to stub the disks because we called Leo list disks to get the list

new version: the list of disks is passed in directly